### ISSUE TITLE
Deployment to AWS S3+Cloudfront fails because of treatment of + in routes

### ISSUE THREAD

**Issue by mwawrusch** [angular/angular-cli#1412]
1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
ALL

2. Versions. Please run `ng --version`. If there's nothing outputted, please

angular-cli: 1.0.0-beta.9
node: 6.1.0
os: darwin x64

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.

Create an app that uses + in route names (per styleguide for future lazy loading)

4. The log given by the failure. Normally this include a stack trace and some
   more information.

Failed to load resource: the server responded with a status of 403 (Forbidden) .../app/+home/home.component.html

5. Mention any other details that might be useful.

Basically when I deploy a production version to S3/Cloudfront the app fails because the templates cannot be resolved, if the path contains a + sign (as per standard for future lazy loading). See this old article: https://forums.aws.amazon.com/thread.jspa?messageID=722673

This might be a temporary problem as hopefully the templates will be compiled into the js (like it is done in the angular-seed problem) but I think it is worth mentioning, given that S3/Cloudfront will be the primary deployment scenario.

**Comment by filipesilva** [comment]
Using the CLI, you can change the lazy loaded route prefix on the `lazyRoutePrefix` property of `angular-cli.json`. 

I understand this is a bigger problem though, so I'm copy pasting this into `angular.io` where style guide issues should be discussed.
